homepage.css

```diff
@@ -53,15 +53,14 @@ nav a:hover{
 ul li ul{
 	padding: 0;
 	position: absolute;
-	top: 458px;
-	left: 1043px;
+	top: 58px;
```

Comment:
Because I have change the link for the nav need to rearrange the the position for the drop down menu link.